src/api_server/swagger/firecracker.yaml

```diff
@@ -753,6 +753,11 @@ definitions:
     properties:
       drive_id:
         type: string
+      ignore_flush:
+        type: boolean
+        description:
+          Makes the device ignore flush requests coming from the guest driver.
+        default: true
```

Comment:
Why not use the `CacheType` enum here instead of this very specific bool?

The enum is forward-looking as it allows us to add options to it or change implementation details for each `CacheType`.
This bool is clearer as it is self-explanatory, but also limited and not extensible.

Reply:
Changed it so that it uses `cache_type`.